### PR TITLE
Add a space on each side of the argument

### PR DESCRIPTION
**Pull request by mune0903**
Hello.
I think there is no unity of space on each side of the arguments.

like this
https://github.com/heartcombo/devise/blob/97aa37bb50c0b7604e49fa3ce594bb85463e28dd/lib/devise/models/confirmable.rb#L337
https://github.com/heartcombo/devise/blob/97aa37bb50c0b7604e49fa3ce594bb85463e28dd/lib/devise/models/confirmable.rb#L378

 So I added a space on each side of the arguments.

### CODE REVIEW

File: lib/devise/controllers/helpers.rb [hunk 1]
```diff
@@ -36,11 +36,11 @@ module ClassMethods
         #     before_action ->{ authenticate_blogger! :admin }  # Redirects to the admin login page
         #     current_blogger :user                             # Preferably returns a User if one is signed in
         #
-        def devise_group(group_name, opts={})
+        def devise_group(group_name, opts = {})
           mappings = "[#{ opts[:contains].map { |m| ":#{m}" }.join(',') }]"
 
           class_eval <<-METHODS, __FILE__, __LINE__ + 1
-            def authenticate_#{group_name}!(favorite=nil, opts={})
+            def authenticate_#{group_name}!(favorite=nil, opts = {})
```
Comment (carlosantoniodasilva) [inline lib/devise/controllers/helpers.rb:43]: Another arg here that could use some additional spacing around `=` :)

**Comment by mune0903** [comment]
> 別の場所があるようですが、それを更新してコミットを確実に押しつぶすことができますか？ありがとう。

Sure!
I'll take care of that.

File: lib/devise/controllers/helpers.rb (hunk 1 above)
Comment (mune0903) [reply lib/devise/controllers/helpers.rb]: Have done all revision!
Could you confirm the commit?

Comment (carlosantoniodasilva) [reply lib/devise/controllers/helpers.rb]: :+1: Merged, thanks!

**Pull request closed without merging**